### PR TITLE
Release/v1.0.1

### PR DESCRIPTION
**Pull request by AlbertPG**
#### Summary of what has been done *

- Prepare the release 1.0.1

#### Reference issues *

There is not any issue since this is a release

## Type of change *
- [ ] New feature (enhancement)
- [ ] Bug fix
- [ ] Documentation update  
- [x] Other (build scripts, metadata, assets, etc.)

## Checklist *
- [x] I have performed a self-review of my own code
- [ ] My code is self-explanatory  
- [x] I have added all new string resources into strings.xml in all languages
- [ ] All possible errors are properly handled  
- [x] I have tested the application with my changes and verify that all tests are passing
- [x] The documentation is up to date

### CODE REVIEW

File: app/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -74,16 +74,20 @@
     <string name="favorite">Favorito</string>
     <string name="recipe_type_hot">Caliente</string>
     <string name="recipe_type_fish">Pescado</string>
+
+    <!-- Favorites fragment -->
+    <string name="no_favorites">No favorites recipes found</string>
```
Comment (santidrj) [inline app/src/main/res/values-es/strings.xml:79]: This line should be translated.

File: app/src/main/res/values-ca/strings.xml [hunk 2]
```diff
@@ -74,16 +74,20 @@
     <string name="recipe_type_spicy">Picant</string>
     <string name="recipe_type_sweet">Dolç</string>
     <string name="recipe_type_sweet_and_sour">Agredolç</string>
+
+    <!-- Favorites fragment -->
+    <string name="no_favorites">No favorites recipes found</string>
```
Comment (santidrj) [inline app/src/main/res/values-ca/strings.xml:79]: This line should be translated.

**Review by santidrj — CHANGES_REQUESTED** [review]
There are some missing translations.

File: app/src/main/res/values-es/strings.xml (hunk 1 above)
Comment (AlbertPG) [reply app/src/main/res/values-es/strings.xml]: Done

File: app/src/main/res/values-ca/strings.xml (hunk 2 above)
Comment (AlbertPG) [reply app/src/main/res/values-ca/strings.xml]: Done

**Review by santidrj — DISMISSED** [review]
Ok

**Review by santidrj — APPROVED** [review]
Ok

**Pull request merged**